components/application_manager/ait.h

```diff
@@ -88,21 +88,21 @@ class Ait
 
     typedef struct
     {
-        uint32_t langCode;
+        uint32_t langCode = 0;
```

Comment:
From the meeting - Marios to revert these initialisation changes and test initialising the instance. Does this pass all HbbTV tests that were failing?